app/app.go

```diff
@@ -704,8 +704,8 @@ func New(
 		return app.mm.RunMigrations(ctx, app.configurator, fromVM)
 	})
 	// this upgrade is for breaking bug fixes on testnet
-	plan0_7_0Rc2HotfixTestnet := "v0.7.0-rc2-hotfix-testnet"
-	app.UpgradeKeeper.SetUpgradeHandler(plan0_7_0Rc2HotfixTestnet, func(ctx sdk.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+	plan0_7_0Rc3HotfixTestnet := "v0.7.0-rc3-hotfix-testnet"
```

Comment:
should it be a new plan instead of replace?

Reply:
I think it's fine, because they'll be released as different versions, a certain version of a binary only need to handle the upgrade plan that upgrades to it.